qbay_test/frontend/test_registration.py

```diff
@@ -0,0 +1,41 @@
+from seleniumbase import BaseCase
+
+from qbay_test.conftest import base_url
+from unittest.mock import patch
+from qbay.models import User
+
+"""
+This file defines all integration tests for the frontend homepage.
+"""
+
+
+class FrontEndHomePageTest(BaseCase):
+
+    def test_login_success(self, *_):
+        """
+        This is a sample front end unit test to login to home page
+        and verify if the tickets are correctly listed.
+        """
+        # open login page
+        self.open(base_url + '/login')
+        # fill email and password
+        self.type("#email", "test0@test.com")
+        self.type("#password", "123456")
+        # click enter button
+        self.click('input[type="submit"]')
+
+        # after clicking on the browser (the line above)
+        # the front-end code is activated
+        # and tries to call get_user function.
+        # The get_user function is supposed to read data from database
+        # and return the value. However, here we only want to test the
+        # front-end, without running the backend logics.
+        # so we patch the backend to return a specific user instance,
+        # rather than running that program. (see @ annotations above)
```

Comment:
Where are the @ annotations?

Reply:
Hi @jsaferrosenthal 

Actually we don't need them any more (no more mocking on the backend) to make things simpler. You can ignore that part.